scipy/_lib/tests/test__util.py

```diff
@@ -378,3 +378,24 @@ def test_contains_nan_2d(self):
 
         data4 = np.array([["1", 2], [3, np.nan]], dtype='object')
         assert _contains_nan(data4)[0]
+
+
+def test__rng_html_rewrite():
+    def mock_str():
+        lines = [
+            'np.random.default_rng(8989843)',
+            'np.random.default_rng(seed)',
+            'np.random.default_rng(0x9a71b21474694f919882289dc1559ca)',
+            ' bob ',
+        ]
+        return lines
+
+    res = _rng_html_rewrite(mock_str)()
+    ref = [
+        'np.random.default_rng()',
+        'np.random.default_rng(seed)',
```

Comment:
So this is intentional for `seed` to be preserved? I suppose it would look funny in an example if a variable `seed` were never used.

Reply:
This was just to show the use of a variable yes. The regex is not looking at the word `seed` per se. The regex is looking at numbers.